samplesheets/templates/samplesheets/irods_request_accept_form.html

```diff
@@ -13,54 +13,56 @@
 
 
 <div class="container-fluid sodar-subtitle-container">
-  <h3>Accept iRODS Delete Request</h3>
+  {% if request_objects|length == 1 %}
+    <h3>Accept iRODS Delete Request</h3>
+  {% else %}
+    <h3>Accept {{ request_objects|length }} iRODS Delete Requests</h3>
+  {% endif %}
 </div>
 
 <div class="container-fluid sodar-page-container">
-  <div class="alert alert-danger">
-    <strong>Warning:</strong> Accepting the request will delete the associated
-    {% if is_collection %}collection{% else %}data object{% endif %} from iRODS.
-    This can <strong>not</strong> be undone without administrator assistance!
-  </div>
-  <div class="alert alert-danger">
-    The following {% if is_collection %}collection{% else %}data object{% endif %} will be deleted: <br />
-    <code class="text-danger">{{ irods_request.path }}</code>
-    {% if affected_objects %}
-      <div class="mt-3 mb-0">
-        The collection to be deleted contains the following data objects:
-        <ul class="mb-0">
-          {% for o in affected_objects %}
-            <li><code class="text-danger">{{ o.path|trim_base_path:irods_request.path }}</code></li>
-          {% endfor %}
-        </ul>
-      </div>
-    {% endif %}
-    {% if affected_collections %}
-      <div class="mt-3 mb-0">
-        The collection to be deleted comprises the following subcollections:
-        <ul class="mb-0">
-          {% for o in affected_collections %}
-            <li><code class="text-danger">{{ o.path|trim_base_path:irods_request.path }}/</code></li>
-          {% endfor %}
-        </ul>
-      </div>
-    {% endif %}
-  </div>
-  <form method="post" enctype="multipart/form-data">
-    {% csrf_token %}
-    {{ form | crispy }}
+  {% if request_objects %}
+    <div class="alert alert-danger">
+      <strong>Warning:</strong> Accepting {% if request_objects|length == 1 %}a request{% else %}requests{% endif %}
+      will delete the associated data objects from iRODS.
+      This can <strong>not</strong> be undone without administrator assistance!
+    </div>
+    <div class="alert alert-danger">
+      <strong>Info:</strong> The following {% if affected_object_paths|length > 1 %}
+      {{ affected_object_paths|length }} data objects
+      {% else %}data object{% endif %} will be deleted:
```

Comment:
I noticed a small bug: collections are also listed as "data objects" here. In the screenshot below, I'm accepting the deletion of the collection `delete-test` which contains 2 data objects. The template lists "3 data objects", one of which is the collection itself. We should only list data objects here, as deleting collections themselves doesn't really matter.

![2023-06_delete_request_bug](https://github.com/bihealth/sodar-server/assets/11211145/b3b5faa0-7b4d-43fa-9655-928a2dd85948)

Reply:
This works fine now, good job fixing it!